cartography/intel/azure/aks.py

```diff
@@ -76,6 +76,7 @@ def get_aks_list(credentials: Credentials, subscription_id: str) -> List[Dict]:
         for aks in aks_list:
             x = aks['id'].split('/')
             aks['resource_group'] = x[x.index('resourcegroups') + 1]
+            aks['publicNetworkAccess'] = aks.get('properties', {}).get('public_network_access', 'Disabled')
```

Comment:
we should set it to false, if there's no value in public_network_access. instead of 'Disabled'